packages/ceramic-account/src/did.ts

```diff
@@ -0,0 +1,33 @@
+import type { CeramicApi, CeramicDocument } from './types'
+
+class DIDDocument {
+  constructor (public ceramicDoc: CeramicDocument, private _ceramic: CeramicApi) { }
+
+  get did (): string {
+    return 'did:3:' + this.ceramicDoc.state.log[0]
+  }
+
+  async linkAccount () { }
```

Comment:
Not sure why this is in here.

Reply:
This is something I'm still working on, but I'm thinking that a class should exclusively own the ceramic documents it represents, and be the only entity writing to them - this might even become enforced by setting them private and exposing the required functionality for fetching info from the document itself. (This `linkAccount` function would be to write the link to the `account` tile in the `did` document, instead of it being done by the `ThreeIDAccount` class)

Reply:
Seems like this should go in the `Account Links Tile` though?